sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -300,15 +300,18 @@ AZ_NODISCARD az_result az_iot_hub_client_telemetry_publish_topic_get(
  * @note C2D MQTT Publish messages will have QoS At Least Once (1).
  *
  * @param[in] client The #az_iot_hub_client to use for this call.
- * @param[in] mqtt_topic_filter An empty #az_span with sufficient capacity to hold the MQTT topic
+ * @param[out] mqtt_topic_filter A char buffer with sufficient capacity to hold the MQTT topic

```

Comment:
What about explicitly mentioning that the output string will be null terminated? It is not understandable from the description that it would be so.

Reply:
updated